### ISSUE TITLE
Request: Custom keyboard shortcuts (at the minimum, for all but the main alt+tab).

### ISSUE THREAD

**Issue by j-a-s-o-n-g** [lokeshgovindu/AltTab#3]
While the typical person might use alt+tab for window navigation (and that's fine), however I am currently having to use AutoHotKey to bind alt+spacebar to the ctrl+alt+tab behavior. It would be nice if you could customize the keyboard shortcuts so I don't have to use AHK.

**Comment by BrommD** [comment]
yes, cos also `backtick` key - on some non english kerybaord layouts - is hard to enter, too.
thx
